python/cudf/cudf/tests/indexes/rangeindex/methods/test_rename.py

```diff
@@ -12,4 +13,4 @@ def test_rename_shallow_copy():
 
     idx = cudf.Index([1])
     result = idx.rename("a")
-    assert idx._column is result._column
+    assert_eq(idx._column, result._column)
```

Comment:
This doesn't necessarily test the same thing anymore. Why did this change?

Reply:
Fixed this test.